src/pages/signup.tsx

```diff
@@ -192,22 +213,25 @@ const SignUpPage = () => {
                         }),
                       ])
                         .then(() => {
-                          router.push("/");
+                          router.push("/verify");
```

Comment:
Add an authorization level to useAuthorization level called "Unverified" that essentially acts like "Unauthenticated" but is for users that need to verify (redirect to /verify)